### ISSUE TITLE
When I run LexI, it has the following error:

### ISSUE THREAD

**Issue by guanzhanyi** [romaonishuk/LexI#1]
When I run LexI, it has the following error:
```
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  51 (X_SetFontPath)
  Value in failed request:  0x3
  Serial number of failed request:  15
  Current serial number in output stream:  16
```
How to solve it?

**Comment by romaonishuk** [comment]
Hi, @guanzhanyi it' because I've hardcoded the path to /resources/fonts in font.hpp :( 
I'll fix this bug today. 
Thanks for the response

**Comment by romaonishuk** [comment]
@guanzhanyi could U, please check again on last master

**Comment by romaonishuk** [comment]
fixed in commit: b4ae1a20cce258f7d17d9ce97bd3d23a2bc0cd1f
